samples/snippets/README.rst

```diff
@@ -0,0 +1,490 @@
+
+.. This file is automatically generated. Do not edit this file directly.
+
+Dialogflow Enterprise Edition  API Python Samples
+===============================================================================
+
+.. image:: https://gstatic.com/cloudssh/images/open-btn.png
+   :target: https://console.cloud.google.com/cloudshell/open?git_repo=https://github.com/GoogleCloudPlatform/python-docs-samples&page=editor&open_in_editor=/README.rst
+
+
+This directory contains samples for Dialogflow Enterprise Edition  API. The `Dialogflow Enterprise Edition API`_ enables you to create conversational experiences across devices and platforms.
+
+
+
+
+.. _Dialogflow Enterprise Edition  API: https://cloud.google.com/dialogflow-enterprise/docs/
+
+
+Setup
+-------------------------------------------------------------------------------
+
+
+
+Authentication
+++++++++++++++
+
+This sample requires you to have authentication setup. Refer to the
+`Authentication Getting Started Guide`_ for instructions on setting up
+credentials for applications.
+
+.. _Authentication Getting Started Guide:
+    https://cloud.google.com/docs/authentication/getting-started
+
+
+
+
+Install Dependencies
+++++++++++++++++++++
+
+#. Clone python-docs-samples and change directory to the sample directory you want to use.
```

Comment:
Note: These repository references are incorrect now. The template issue is tracked in https://github.com/googleapis/synthtool/issues/867.